### ISSUE TITLE
Fonts are completely broken

### ISSUE THREAD

**Issue by trappar** [exceljs/exceljs#870]
Currently the first declared font will affect the entire worksheet. It doesn't seem to matter if you set a column, row, or cell - all of these will affect the whole document with seemingly no way to override it once it's been set.

It also seems like subsequently declared fonts affect future cells. Are fonts meant to be some sort of stateful thing? It seems like they should be reset unless they have been explicitly declared.

Here's a simple script to reproduce this issue:

```javascript
const Excel = require('exceljs');

const wb = new Excel.Workbook();
const ws = wb.addWorksheet('My Sheet');

ws.columns = [
    {header: 'First', key: 'first', width: 10},
    {header: 'Second', key: 'second', width: 10},
];

const a2 = ws.getCell('A2');
const b2 = ws.getCell('B2');

a2.value = 'a2 test';
b2.value = 'b2 test';

a2.font = {
    name: 'Comic Sans MS',
    family: 4,
    size: 16,
    underline: true,
    bold: true
};

b2.font = {
    name: 'Arial Black',
    color: {argb: 'FF00FF00'},
    family: 2,
    size: 14,
    italic: true
};

wb.xlsx.writeFile(__dirname + '/test.xlsx');
```

**Comment by DimosthenisK** [comment]
I am having the same issue. Also, this is listed as fixed in a merged PR 7 days ago.

**Comment by tianhai82** [comment]
I also encounter the same issue on 1.12.2.

**Comment by hzamir** [comment]
Encountering this issue also

**Comment by kingofferelden** [comment]
Encountered this issue too

**Comment by DimosthenisK** [comment]
Guys, this is really important, can we get it fixed? I tried finding out and submitting a pull request but I can't find what is going on.

**Comment by nfunwigabga** [comment]
Same problem here

**Comment by havietduc91** [comment]
Same issue here, cell font styling is applied for all cells 
I'm trying set font for cell A1, something like this
```
 worksheet.getCell('A1').font = {
          name: 'Comic Sans MS',
          family: 4,
          size: 16,
          underline: true,
          bold: true
        };
```
But all cells have the same font with A1 cell, anyone has any solution to resolve it?

**Comment by havietduc91** [comment]
+ 1 
We can down version to resolve this issue, the exact version is `"exceljs": "1.12.0"`
I try it and it's work pretty perfectly

**Comment by autukill** [comment]
https://github.com/exceljs/exceljs/commit/3cf69cd8b72b69de85556fdeef5705e74482288a

**Comment by hzamir** [comment]
supposedly fixed today 1.14.0 found it 15 minutes after it was published.
